### PR TITLE
Add postgresql_cluster_member entity type

### PR DESCRIPTION
**Pull request by a1exsh**
For consistent naming, remove namespace from postgresql_cluster entity id,
since it is already part of service DNS name and add a pg- prefix, e.g:
pg-mypostgres.default.svc.cluster.local

The cluster member entities are created from pods of application 'spilo' with
the same 'version' label as the service used to create cluster entity has.

### CODE REVIEW

**Comment by Jan-M** [comment]
:+1:

File: zmon_agent/discovery/kubernetes/cluster.py [hunk 1]
```diff
@@ -476,6 +480,42 @@ def get_postgresql_clusters(kube_client, cluster_id, alias, region, infrastructu
...
+        obj = pod.obj
+
+        # TODO: filter in the API call
+        labels = obj['metadata'].get('labels', {})
+        if labels.get('application') != 'spilo' or labels.get('version') is None:
+            continue
+
+        pod_number = pod.name.split('-')[-1]
+        pod_namespace = obj['metadata']['namespace']
+        service_dns_name = '{}.{}.svc.cluster.local'.format(labels['version'], pod_namespace)
```
Comment (mohabusama) [inline zmon_agent/discovery/kubernetes/cluster.py:542]: is empty string version ok in this case?

Comment (a1exsh) [reply zmon_agent/discovery/kubernetes/cluster.py:542]: Yeah, probably not.  We could check if it's something that evaluates to `True` above.  But if it's not empty, but wrong--that we cannot prove.

**Comment by Jan-M** [comment]
:+1:

File: zmon_agent/discovery/kubernetes/cluster.py (hunk 1 above)
Comment (mohabusama) [reply zmon_agent/discovery/kubernetes/cluster.py:542]: @a1exsh are you going to change it or shall we merge it with the current implementation?

Comment (a1exsh) [reply zmon_agent/discovery/kubernetes/cluster.py:542]: I don't think an extra check here adds any value: an empty string is as bad as any other incorrect value, and that we can't check.

**Comment by mohabusama** [comment]
👍

**Comment by elauria** [comment]
👍

**Pull request merged**